tests/test_reverse.py

```diff
@@ -0,0 +1,27 @@
+import unittest
+
+import numpy as np
+
+from audiomentations import Reverse
```

Comment:
When running `pytest`, I get the following error:

```______________________________________ ERROR collecting tests/test_reverse.py ______________________________________
ImportError while importing test module 'C:\Users\Iver\Code\audiomentations\tests\test_reverse.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests\test_reverse.py:5: in <module>
    from audiomentations import Reverse
E   ImportError: cannot import name 'Reverse' from 'audiomentations' (C:\Users\Iver\Code\audiomentations\audiomentations\__init__.py)
```

To fix this error, either change the import here or add `Reverse` to the list of publicly exposed transforms in audiomentations/\_\_init\_\_.py

Reply:
Done.